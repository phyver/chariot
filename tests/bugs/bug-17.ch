codata t where D1 : t -> 'x

val a : t
  | a = D1 # a
