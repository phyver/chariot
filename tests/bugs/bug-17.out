>> option verbose is set to 1
** error: type parameter x is free
